docs/usage/configuration-options.md

```diff
@@ -2501,13 +2501,10 @@ image: my.new.registry/aRepository/andImage:1.21-alpine
 ## registryAliases
 
 You can use the `registryAliases` object to set registry aliases.
-This feature only works with these managers:
 
-- `helm-requirements`
-- `helmv3`
-- `helmfile`
+The aliases are interpreted by the relevant manager.
 
-The managers listed above all have this default registryAlias:
+The `helm-requirements`, `helmv3` and `helmfile` managers listed above all have this default registryAlias:
```

Comment:
```suggestion
The `helm-requirements`, `helmv3` and `helmfile` managers have this default registryAlias:
```

We don't actually _list_ the managers anymore.

Reply:
Oops, thanks!